docs/user/config-file/v2.rst

```diff
@@ -468,12 +468,10 @@ When ``build.commands`` is used, none of the :term:`pre-defined build jobs` will
 This allows you to run custom commands and control the build process completely.
 The ``$READTHEDOCS_OUTPUT/html`` directory will be uploaded and hosted by Read the Docs.
 
-.. warning::
+.. note::
 
-   This feature is in a *beta phase* and could suffer incompatible changes or even removed completely in the near feature.
-   We are currently testing `the new addons integrations we are building <rtd-blog:addons-flyout-menu-beta>`_
+   We are currently testing :ref:`the new addons integrations we are building <rtd-blog:addons-flyout-menu-beta>`
```

Comment:
Probably use the same copy as above here. We should be promoting addons a bit here :)